packages/shorebird_cli/lib/src/commands/releases/list_releases_command.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:barbecue/barbecue.dart';
+import 'package:mason_logger/mason_logger.dart';
+import 'package:shorebird_cli/src/auth_logger_mixin.dart';
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/shorebird_config_mixin.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+
+/// {@template list_releases_command}
+///
+/// `shorebird releases list`
+/// List all releases for this app.
+/// {@endtemplate}
+class ListReleasesCommand extends ShorebirdCommand
+    with AuthLoggerMixin, ShorebirdConfigMixin {
+  /// {@macro list_releases_command}
+  ListReleasesCommand({
+    required super.logger,
+    super.auth,
+    super.buildCodePushClient,
+  });
+
+  @override
+  String get name => 'list';
+
+  @override
+  String get description => 'List all releases for this app.';
+
+  @override
+  Future<int> run() async {
+    if (!auth.isAuthenticated) {
+      printNeedsAuthInstructions();
+      return ExitCode.noUser.code;
+    }
+
+    if (!hasShorebirdYaml) {
+      logger.err(
+        '''Shorebird is not initialized. Did you run ${lightCyan.wrap('shorebird init')}?''',
+      );
+      return ExitCode.config.code;
+    }
+
+    final appId = getShorebirdYaml()!.appId;
```

Comment:
```suggestion
    final appId = getShorebirdYaml()!.appId.value!;
```
so we don't need to force unwrap a bunch of times later on

Reply:
Done